### ISSUE TITLE
[Adaptive Card]: Elements marked as presentational should not have global ARIA or tabindex to ensure all screen readers ignore them.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#8362]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS version: 22H2 (Build 25314.1000)
Browser: Edge Dev (Version 111.0.1661.36 (Official build) dev (64-bit))
URL: https://adaptivecards.io/blog/
Tool: Accessibility Insights for Web

**Repro Steps:**

Open the above URL in edge dev browser.
Turn on the fast pass of accessibility insights for web.
Observe the issue.

**Actual Result:**
Elements marked as presentational have global ARIA or tabindex.

**Expected Result:**
Elements marked as presentational should not have global ARIA or tabindex to ensure all screen readers ignore them.

**Snippet**: <a href="/blog/2022/Community-Call-March/" class="coverImage" style="background-image: url('https://techcommunity.microsoft.com/t5/image/serverpage/image-id/356326i03254E96BDFD72C2/image-size/large?v=v2&amp;px=999')" role="presentation" aria-label="📣 Community Call - March, 2022"></a>

**How to fix:** 
Fix all of the following:
1.   Element is not focusable.
2.   Element does not have global ARIA attribute

**User Impact:**
Screen reader user will get impacted as user will have difficulty in making track or in navigating the content properly.

**MAS Reference link:**
https://microsoft.sharepoint.com/:w:/s/accessibility/EXdDUZzldv5MgwDaMHnTzcMBcT8opKGncYjU-gCJ4uqQdg?e=9nuVmt

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

<img width="960" alt="MAS1 3 1 Ensures marked as presentational" src="https://user-images.githubusercontent.com/104483005/224677067-e36ab1ad-871d-4ca2-8355-815818dfdf86.png">

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Hi @anna-dingler,

As this issue is still repro, so could you please let us know is there any future plan to fix this issue?

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/51f743e1-d37c-4327-bc46-09b9051a5826)
